src/lib/Scenes/Favorites/Components/Shows/index.tsx

```diff
@@ -43,7 +43,7 @@ export class Shows extends Component<Props, State> {
     const shows = this.props.me.followsAndSaves.shows.edges.map(edge => edge.node)
 
     if (shows.length === 0 || !shows) {
-      return <ZeroState title="You haven’t followed any shows yet" />
+      return <ZeroState title="You haven’t followed any shows yet" subtitle="" />
```

Comment:
Is this needed?

Reply:
Na, I was getting an error locally but I removed it